.github/workflows/ci.yml

```
@@ -1,122 +1,68 @@
-name: CI - Build Checks
+name: CI Pipeline
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)